migrations/0018_create_rsvps.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS rsvps (
+    id INTEGER PRIMARY KEY NOT NULL,
+    user_id INTEGER NOT NULL,
+    event_id INTEGER NOT NULL,
+    ticket_id INTEGER NOT NULL,
+    transaction_id INTEGER,
+    price INTEGER,
+
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    checkin_at TIMESTAMP
```

Comment:
how will this be recorded? is there gonna be an electronic system for checkins eventually?